File: reorg/mv_files.pl

open IN, "two.lst";
while ($line=<IN>){
#print $line;
chomp($line);
@words=split(/\s+/,$line);
$ren=$words[3];
$ren=~ s/^orig//;
print "/Users/george/s5cmd/s5cmd mv s3://idc-open-data-two-logs/$words[3] s3://idc-open-data-two-logs/orig/$ren\n";
}

File: reorg/mv_files.py
bucket="s3://idc-open-data-logs"
months={5:10}

mlist=list(months.keys())
for mnth in mlist:
  numd= months[mnth]
  for i in range(1,numd+1):
    #print(str(mnth)+' '+str(i))
    src=bucket+'/2023-'+str(mnth).zfill(2)+'-'+str(i).zfill(2)+'*'
    dest=bucket+'/orig/'
    print("echo moving "+src)
    print("/home/ec2-user/s5cmd mv '"+src+"' '"+dest+"'")



